pkg/resourcemanager/apis/config/validation/validation.go

```diff
@@ -200,3 +201,12 @@ func validateSyncPeriod(val *metav1.Duration, fldPath *field.Path) field.ErrorLi
 
 	return allErrs
 }
+
+// ValidateSystemComponentsConfigWebhookConfig validates the given `SystemComponentsConfigWebhookConfig`.
+func ValidateSystemComponentsConfigWebhookConfig(conf *config.SystemComponentsConfigWebhookConfig) field.ErrorList {
```

Comment:
I assume the intention was to call this func from some of the above validation funcs (for example `validateResourceManagerWebhookConfiguration`). The newly introduced func seems unused currently.

We can unexport it, seems that it will be used only from this pkg.
```suggestion
func validateSystemComponentsConfigWebhookConfig(conf *config.SystemComponentsConfigWebhookConfig) field.ErrorList {
```